include/ui/form.php

```diff
@@ -204,6 +204,30 @@ function bd_render_post_type_dropdown( $slug ) {
 <?php
 }
 
+/**
```

Comment:
Move the contents of this function into the `render_user_role_dropdown` function that you created in the `Renderer` class itself.